apps/client/src/app/pages/map/route/hooks/useCalculateTravelTime.tsx

```diff
@@ -0,0 +1,30 @@
+import { LngLat } from 'mapbox-gl';
+
+const calculateTravelTime = async (
+  start: LngLat,
+
+  end: LngLat
+): Promise<number> => {
+  const response = await fetch(
+    `https://api.mapbox.com/directions/v5/mapbox/cycling/${start.lat},${
```

Comment:
Does the application only calculate the travel time in cycling? please try to create a enum for this